packages/core/bootstrap/src/lib/ws/epics.ts

```diff
@@ -332,12 +335,15 @@ export const metricsEpic: Epic<AnyAction, AnyAction, any, any> = (action$, state
           break
         case subscriptionError.type:
           ws_subscription_errors.labels(subscriptionErrorLabels(action.payload)).inc()
-          logger.error('WS: subscription error', { payload: action.payload })
+          logger.error(action.payload.reason, { payload: action.payload })
```

Comment:
Hmm, I'd rather we revert this so its consistent with other logging, and the `reason` should be in the payload, right?

Reply:
Yep ok